+nosnoc/+dcs/Gcs.m

```diff
@@ -0,0 +1,67 @@
+classdef Gcs < nosnoc.dcs.Base
+    properties
+        lambda
+
+        f_x
+
+        dims
+
+        f_fun
```

Comment:
i find it like on other similar occasion unclear, 
f_x is  f_unconstrained + nabla_c'*lambda;
and f_fun is made of f_unconstrained? more instructive names (and possibly comments would make it more sustainable)